frontend/src/metabase/components/LoadingAndErrorWrapper/DelayedLoadingAndErrorWrapper.tsx

```diff
@@ -56,7 +53,9 @@ export const DelayedLoadingAndErrorWrapper = ({
     return <>{children}</>;
   }
   if (!showWrapper) {
-    return <>{blankComponent}</>;
+    // make tests aware that things are loading
+    const blankComponent = <span data-testid="loading-spinner" />;
+    return blankComponent;
```

Comment:
Can we just return the value here?
```suggestion
    return <span data-testid="loading-spinner" />;
```

Reply:
c1ff042f245160635eb04918a36a8611a10011b9